packages/core/src/theme/Theme.ts

```diff
@@ -28,25 +28,26 @@ export class Theme {
     this.id = id;
     this.name = name;
   }
+}
 
-  getCharacteristicValue(
-    characteristicName: characteristic,
```

Comment:
Don't we need to pass the mode also here?

Reply:
The characteristics themselves are not mode dependent.